src/XML/src/FMI3/fmi3_xml_variable.c

```diff
@@ -1129,15 +1173,17 @@ int fmi3_xml_handle_IntXXVariable(fmi3_xml_parser_context_t* context, const char
     res = fmi3_xml_handle_Variable(context, data);
     if (res) return res;
 
-    if(!data) {
-        fmi3_xml_model_description_t* md = context->modelDescription;
-        fmi3_xml_type_definitions_t* td = &md->typeDefinitions;
-        fmi3_xml_variable_t* variable = jm_vector_get_last(jm_named_ptr)(&md->variablesByName).ptr;
-        fmi3_xml_variable_type_base_t * declaredType = 0;
-        fmi3_xml_int_type_props_t * type = 0;
-        int hasStart;
+    md = context->modelDescription;
+    td = &md->typeDefinitions;
+    variable = jm_vector_get_last(jm_named_ptr)(&md->variablesByName).ptr;
+    type = 0;
 
-        declaredType = fmi3_get_declared_type(context, elmID, &defaultType->super) ;
+    if(!data) {
+        fmi3_xml_variable_type_base_t* declaredType = fmi3_get_declared_type(
+            context,
+            elmID,
+            &defaultType->super
+        );
```

Comment:
I suggest we don't create a new line for each argument, but just do double indentation on next line. That's how we do it almost everywhere else. Unless it greatly increases readability that is.

Reply:
Fixed in 7465def